src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -337,6 +342,26 @@ impl LocalStateStore for LocalHummockStorage {
         .await
     }
 
+    async fn try_flush(&mut self) -> StorageResult<()> {
+        if self.mem_table.kv_size.size() > MEM_TABLE_SPILL_THRESHOLD {
+            tracing::info!(
+                "The size of mem table exceeds {} Mb and spill occurs. table_id {}",
+                MEM_TABLE_SPILL_THRESHOLD >> 20,
+                self.table_id.table_id()
+            );
+
+            if self.spill_offset < AVAILABLE {
```

Comment:
Better to increment the spill_offset in `flush`? In the current implementation, the `flush` call in barrier is the same as the last `flush` triggered by `try_flush` if there is any.